webserver/templates/index/current-status.html

```diff
@@ -0,0 +1,60 @@
+{%- extends 'base.html' -%}
+{%- block title -%}Status - ListenBrainz{%- endblock -%}
+{%- block content -%}
+
+  <h2 class="page-title">Current status</h2>
+
+  <div class="row">
+
+    <div class="col-md-7 col-lg-8">
+      <h3>Incoming listens</h3>
+
+      <p>
+        Stats for our kafka listens stream: 
+      </p>
+
+      <table class="table table-bordered table-striped">
+          <tr>
+              <td>offset</td>
```

Comment:
I tried to describe them last night, but wasn't able to find good ways of doing that. Partially because I don't know enough about kafka to do a good job. Good enough for now, I say.